barrels/LogMonkey/Logger.mc

```diff
@@ -0,0 +1,172 @@
+using Toybox.Time.Gregorian as Greg;
+using Toybox.Lang as Lang;
+
+module LogMonkey {
+
+    //! The Logger class provides public APIs to log various types of
+    //! messages. Each Logger class is associated with a log level which
+    //! is set within the Logger.initialize() function.
+    class Logger {
+
+        private var mLogLevel, mLogStream;
```

Comment:
Typical coding style guidelines would require each declaration to be on its own line.

Reply:
Done